internal/plugins/manifests/init.go

```diff
@@ -18,12 +18,17 @@ package manifests
 import (
```

Comment:
These changes shouldn't be in the manifests plugin, they should be in their own `internal/plugins/envtest` plugin.